packages/mongodb-rag-ingest/src/commands/all.ts

```diff
@@ -10,14 +10,21 @@ import {
 } from "../withConfig";
 
 const commandModule: CommandModule<unknown, LoadConfigArgs> = {
-  command: "all",
+  command: "all [permanentlyDeletePages]",
```

Comment:
```suggestion
  command: "all",
```

i'm reasonably sure that this can be deleted b/c of the below use of 
```ts
.option("permanentDeletePages", ...)
```